### PR TITLE
Rds public exposure

### PR DESCRIPTION
**Pull request by GauravChawda**

### CODE REVIEW

File: cartography/intel/aws/rds.py [hunk 1]
```diff
@@ -594,6 +609,34 @@ def load_rds_snapshots(
...
+def load_rds_snapshot_attributes(neo4j_session: neo4j.Session, data: Dict, aws_update_tag: int,) -> None:
+    """
+    Ingest the RDS snapshot attributes to neo4j and link them to RDS snapshot.
+    """
+    ingest_rds_snapshot_attributes = """
+    UNWIND $snapshotAttributes as result
+        MATCH (rdsSnapshot:RDSSnapshot{db_snapshot_identifier: result.DBSnapshotIdentifier})
+        WITH result, rdsSnapshot
+        UNWIND result.DBSnapshotAttributes as attribute
+            MERGE (attrib:RDSSnapshotAttribute{name: attribute.AttributeName})
```
Comment (ganeshjagdale9545) [inline cartography/intel/aws/rds.py:622]: Implement cleanup job for this

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
one minor change

**Review by ganeshjagdale9545 — APPROVED** [review]
looks good

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**